Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/GhostGuards.dfy

```diff
@@ -1,5 +1,5 @@
 // NONUNIFORM: need to add support for tooltips to the new CLI (if that makes sense)
```

Comment:
Is this comment obsolete now?

Reply:
The message after NONUNIFORM is obsolete yes.